test/e2e/storage/external/external.go

```diff
@@ -302,6 +309,34 @@ func (d *driverDefinition) GetDynamicProvisionStorageClass(e2econfig *testsuites
 	return testsuites.GetStorageClass(sc.Provisioner, sc.Parameters, sc.VolumeBindingMode, f.Namespace.Name, "e2e-sc")
 }
 
+func (d *driverDefinition) GetTimeouts() *framework.TimeoutContext {
+	timeouts := framework.NewTimeoutContextWithDefaults()
+	if d.Timeouts != nil {
```

Comment:
Can we instead check if d.Timeouts == nil { return timeouts} and avoid the indent.